### ISSUE TITLE
Create a static image widget

### ISSUE THREAD

**Issue by JanLenoch** [Kentico/xperience-training-13#62]
### Who

- [ ] Anonymous visitor
- [ ] Patient
- [ ] Doctor
- [x] Content editor
- [ ] Marketer
- [ ] Administrator
- [ ] Developer

### Needs what

To insert still images into pages.

### Why

Why is the feature needed?

### Issue expands upon

This issue expands upon another issue(s): #`issue number`

### Blockers

This issue is blocked by the following issue(s): #`issue number`

**Comment by JanLenoch** [comment]
Resolved via 96336883430ba806f6d75636cd7bc4437e87d101
